bundles/org.openhab.binding.astro/src/test/java/org/openhab/binding/astro/TestUtils.java

```diff
@@ -0,0 +1,56 @@
+/**

```

Comment:
Can you move this file into the `internal` package? Or `internal.utils`?

Reply:
We already have **org.openhab.binding.astro.internal.util** packgage under **src/main/java**. However the best place for this method maybe would be **DateTimeUtils** class, but the only one method **org.openhab.binding.astro.TestUtils.newCalendar(int, int, int, int, int, TimeZone)** is used only in unit tests, so I have created a separate class for it.

Maybe it would be good to merge **TestUtils** into **DateTimeUtils**? What do you think?

Reply:
Yes, makes sense - if you expect to make further use of it in the future. Otherwise we can merge the method into `SunCalcTest` too.

Reply:
For now it is used only in tests, so at the moment it make sense to move it to ```SunCalcTest```. We can easy refactor it later; if required.